simtools/applications/calculate_trigger_rate.py

```diff
@@ -62,6 +62,12 @@ def _parse(label, description):
         action="store_true",
     )
 
+    config.parser.add_argument(
+        "--rht",
```

Comment:
This requires internal knowledge on how rht works. Do we want to call it according to the actual way the trigger rate is calculate, using the mean of the distributions and its expected upper edge? It will be hard to see otherwise in a couple of months what is meant here.

Reply:
I added the explanation to the module and added now here too. I think it is important to keep the same nomenclature such that it is clear that we expect the same results either using this tool or using the rht tool. It increases the reliability of this newly developed tool.

Reply:
Not convinced. Nobody except Konrad and you will remember what rht is. Do I see it correctly that the only difference is the calculation of the radius of the scatter area? I think it would be clearer to any user to name it accordingly, so that no historical knowledge is required.

Reply:
historical knowledge is the whole idea here, as explained above.

Reply:
I don't understand: how would anyone know about rht outside of this small circle? So either  we remove the functionality or we describe it without knowledge of script on Konrad's laptop. Let me know if you need help with.

Reply:
I removed this option.